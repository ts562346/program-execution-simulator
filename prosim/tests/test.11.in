test1 7
LOOP 4
  DOOP 2
END
LOOP 3
  BLOCK 3
END
HALT

