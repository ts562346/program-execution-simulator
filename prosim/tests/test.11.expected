00000: DOOP
00002: DOOP
00004: DOOP
00006: DOOP
00008: BLOCK
00011: BLOCK
00014: BLOCK
00017: HALT

DOOP count : 4
DOOP time  : 8
BLOCK count: 3
BLOCK time : 9
